clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinatorTest.java

```diff
@@ -1193,6 +1215,12 @@ public void testHandleLeaveGroupResponseWithException() {
     }
 
     private RequestFuture<Void> setupLeaveGroup(LeaveGroupResponse leaveGroupResponse) {
+        return setupLeaveGroup(leaveGroupResponse, "test maybe leave group", "test maybe leave group");
+    }
+
+    private RequestFuture<Void> setupLeaveGroup(LeaveGroupResponse leaveGroupResponse,
+                                                String expectedLeaveReason,
+                                                String actualLeaveReason) {
```

Comment:
nit: I would put `actualLeaveReason` first. It is a bit more natural. Should we call it `leaveReason`?